wiki/How_You_Can_Help!/en.md

```diff
@@ -1,77 +1,77 @@
-<!-- TODO: tons of things to add here. might be related to Projects article -->
+<!-- TODO: tons of things to add here. might be related to Projects articl -->
 
 # How you can help!
 
-osu!, like any other game, has a very vibrant community, composed of people from all around the world, with differing skill sets and different purposes.
+osu! has a very vibrant community composed of people from all around the world. And anyone can be a part of the community in their own way! 
 
-Anyone can be a part of the community in their own way. Here are a few examples to start with:
+And there are many ways you can help out in the community:
```

Comment:
Short-handing this makes the tone of this very vague and you seem to imply these are the *only* areas you can help out.

Reply:
Yeah you're right; I'll reword it someone tomorrow afternoon.